### DISCUSSION TITLE
How to configure reporting for aqaraOpple on Xiaomi SSM-U01?

### DISCUSSION THREAD

**Discussion by git1106** [Koenkk/zigbee2mqtt#12269] [Devices]
I'd like to use the energy consumption measurements on my [Xiaomi SSM-U01](https://www.zigbee2mqtt.io/devices/SSM-U01.html).

Being new to Zigbee I grepped around in the source and searched on the Web and came up that I need to configure reporting

```
$ mosquitto_pub -t zigbee2mqtt/bridge/request/device/configure_reporting -m '{"id":"SSM-U01","cluster":"seMetering","attribute":"currentSummDelivered","minimum_report_interval":294,"maximum_report_interval":300,"reportable_change":0}
$ mosquitto_pub -t zigbee2mqtt/bridge/request/device/configure_reporting -m '{"id":"SSM-U01","cluster":"haElectricalMeasurement","attribute":"activePower","minimum_report_interval":295,"maximum_report_interval":300,"reportable_change":0}'
```
(The same is also in the frontend)

However, no values were exposed to MQTT although I can see in the debug messages that Zigbee messages arrive with those attributes.

Grepping a bit more in the source brought me to #10709, which says that the `seMetering` is broken and ignored by zigbee2mqtt. Instead `aqaraOpple` should be used.

However, I can not find for what attribute I have to turn on reporting.  The frontend offers only 2 attributes for `aqaraOpple`: `mode` and `illuminance`. The latter seems inappropriate for a switch. `mode` reports the value `command`, but nothing else.

> Zigbee2MQTT version  [1.25.0](https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.25.0) commit: [6f1460e](https://github.com/Koenkk/zigbee2mqtt/commit/6f1460e)
> Coordinator type ConBee2/RaspBee2

**Comment by nimai** [comment]
Hello @git1106, did you find a solution to your problem?
I have the exact same issue, and I came to the same conclusion that I should use aqaraOpple, but I can't find any hint on which attribute to use. From the mentioned ticket, I understand that repairing the switch should reconfigure it correctly, but it doesn't in my case.
I also have the latest firmware on the devices.
I'm starting to wonder if there's not an incompatibility with Raspbee2 coordinator, because I also happen to have that one too, which is not the case of anyone in the conversation on #10709.

**Reply by git1106** [reply]
No, I have not found a solution. I believe the values seen in the log messages are correct. I don't know any better solution than parsing the log file.

I have not used my meter recently, so I have no fresh information.
